### PR TITLE
Add Polish translations

### PR DESCRIPTION
**Pull request by lunarna-gh**
Translated all of the pages. Please let me know if there is anything left to translate that I missed.

### CODE REVIEW

File: content/donate/index.pl.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+{{ donate_buttons() }}
+
+Cykliczne wpłaty to najlepszy sposób na zapewnienie względnie stabilnego dochodu dla
+projektu i zmotywowanie nas do realizacji długoterminowych zadań i celów. Ale możesz też
+wpłacić jednorazowo.
+
+### Przelew bankowy
+
+{{ bank_transfer(lang='en') }}
```
Comment (biodranik) [inline content/donate/index.pl.md:59]:
```suggestion
{{ bank_transfer(lang='pl') }}
```

File: content/donate/index.pl.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+### Krypto
+
+Proszę rozważyć coroczną dotację, aby zmniejszyć dodatkowe opłaty. Algorand, Tron,
+sieci Litecoin i Monero oferują niskie opłaty, ale sytuacja zmienia się każdego dnia.
+
+{{ crypto_table(lang='en') }}
```
Comment (biodranik) [inline content/donate/index.pl.md:66]:
```suggestion
{{ crypto_table(lang='pl') }}
```

File: content/_index.pl.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+- Brak denerwującej rejestracji
+- Brak obowiązkowych poradników
+- Brak hałaśliwego spamu na e-mail
+- Brak powiadomień
+- Brak niepotrzebnych funkcji
+- ~~Brak pestycydów~~ Czysto organiczne!
+
+Ta aplikacja jest zweryfikowana przez <a href='https://reports.exodus-privacy.eu.org/en/reports/app.organicmaps/latest/'>Exodus Privacy Project</a>:
+<br/>
+<img src='images/privacy/exodus.png' width='400'>
```
Comment (biodranik) [inline content/_index.pl.md:59]:
```suggestion
<img src='/images/privacy/exodus.png' width='400'>
```

File: content/_index.pl.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+Organic Maps nie żąda nadmiernych uprawnień by Cię szpiegować:
+
+<img src='images/privacy/om.jpg' width='400'>
```
Comment (biodranik) [inline content/_index.pl.md:63]:
```suggestion
<img src='/images/privacy/om.jpg' width='400'>
```

File: content/_index.pl.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+<img src='images/privacy/mm.jpg' width='400'>
```
Comment (biodranik) [inline content/_index.pl.md:64]:
```suggestion
<img src='/images/privacy/mm.jpg' width='400'>
```

**Review by biodranik — COMMENTED** [review]
Thank you! If possible, please also run `npm i && npm run format` to cleanup the markdown. If not, that's OK, I can help later.

**Comment by lunarna-gh** [comment]
I ran the command and fixed the issues.

**Comment by lunarna-gh** [comment]
Actually, I noticed the listing on the app stores has been recently translated using Weblate. Please don't merge this for a second while I make the terminology more consistent

**Review by biodranik — APPROVED** [review]
Bardzo dziękujemy za Twój wkład! Jesteśmy naprawdę wdzięczni.

**Pull request merged**